src/loaders/OmeZarrLoader.ts

```diff
@@ -147,6 +148,7 @@ class SmartStoreWrapper implements AsyncStore<ArrayBuffer, RequestInit> {
     if (this.requestQueue) {
       return this.requestQueue.addRequest(key, () => this.getAndCacheItem(item, key, opts));
     } else {
+      // Should we ever hit this code?  We should always have a request queue.
```

Comment:
Currently, we don't, since the zarr loader builds its own queue. In the future, we could end up sharing a queue by passing it into loader constructors, as currently happens with the cache. The cache is currently optional and null-checked, since we can get on just fine without it, and since accepting it via the constructor means we rely on the client to build one themselves first. This is following the same logic.